src/QMCDrivers/RMC/RMCUpdatePbyP.cpp

```diff
@@ -411,7 +410,7 @@ namespace qmcplusplus
       //Collectables should be evaluated on center bead.  Here we go.
       Walker_t& centerbead = W.reptile->getCenter();
       W.loadWalker(centerbead,true);
-      W.update(false);  //Called to recompute S(k) and distance tables.  Recompute S(k)?  False.
+      W.update();  //Called to recompute S(k) and distance tables.  Recompute S(k)?  False.
```

Comment:
Recompute is actually true. skipSK is false.

Reply:
I agree.  I don't know why I said "recompute" in the comment.  Fixed.